mlrun/api/schemas/feature_store.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+from typing import Optional, List
+
+from pydantic import BaseModel
+
+
+class Feature(BaseModel):
+    name: str
+    value_type: str
+
+    class Config:
+        orm_mode = True
+
+
+class FeatureSetMetaData(BaseModel):
+    name: str
+    tag: Optional[str]
+    labels: Optional[dict]
+    updated: Optional[datetime]
+    uid: Optional[str]
+
+
+class FeatureSetSpec(BaseModel):
+    entities: List[Feature]
+    features: List[Feature]
+
+
+class FeatureSet(BaseModel):
+    metadata: FeatureSetMetaData
+    spec: FeatureSetSpec
+    status: Optional[dict]
+
+
+class FeatureSetUpdate(BaseModel):
+    features: Optional[List[Feature]]
+    entities: Optional[List[Feature]]
+    status: Optional[dict]
+    labels: Optional[dict]
+
+
+# state is extracted from the full status dict to enable queries
```

Comment:
move comment to be above the state field

Reply:
👍